### PR TITLE
Feature - Search favourite songs

### PR DESCRIPTION
**Pull request by cesarandex**
- Added button to search user's most played songs for the last 6 months
- Removed translate util
- Reworked SearchSongs expanding logic for code clean-up (open to suggestions)
- Added message for no active device

### CODE REVIEW

File: src/pages/Home/SearchSongs/SearchSongs.js [hunk 1]
```diff
@@ -5,67 +5,75 @@ import { useIntl } from 'react-intl';
...
   const dispatch = useDispatch();
 
   const searchSongs = async (query, newOffset = offset) => {
     const result = await search(query, newOffset);
+    setExpanded(true);
     setSearchResults(result);
     setSearchTimeout(null);
   };
 
+  const favouriteSongs = async (newOffset = offset) => {
```
Comment (bonavida) [inline src/pages/Home/SearchSongs/SearchSongs.js]: I'd suggest to use verbs for method naming. For example, `getFavouriteSongs` here and `getTopTracks` in the line below.

File: src/services/spotify.js [hunk 2]
```diff
@@ -48,3 +49,12 @@ export const search = (query, offset) => {
     .get(endpointUrlSearch, { params })
     .then(res => res.data.tracks.items);
 };
+
+export const topTracks = (offset, time_range = 'medium_term') => {
```
Comment (bonavida) [inline src/services/spotify.js]: We can use constants also for the time range.

File: src/pages/Home/SearchSongs/SearchSongs.js (hunk 1 above)
Comment (cesarandex) [reply src/pages/Home/SearchSongs/SearchSongs.js]: Agree, done

File: src/services/spotify.js (hunk 2 above)
Comment (cesarandex) [reply src/services/spotify.js]: Yep, had that in mind as well just didn't initially created the constants as we don't have a selector for the time range yet. Added it still

**Pull request merged**